scripts/scr_add_man/scr_add_man.gml

```diff
@@ -35,12 +35,63 @@ function scr_add_man(man_role, target_company, spawn_exp, spawn_name, corruption
 		if (other_gear = true) {
 	// Factions 1-5 are part of Imperial family
 		// Faction 1 - Space Marine
+			/* TODO - add the right faction stuff, and move the "switch (man_role) {" before uncommenting
+			case "Chapter Servitor":
+				obj_ini.wep1[target_company][good] = "Combat Knife"; TODO - consider creating a melee mechadendrite instead?
+				obj_ini.wep2[target_company][good] = "";
+				obj_ini.armour[target_company][good] = "Scout Armour";
+				obj_ini.mobi[target_company][good] = "Servo-arm";
+				obj_ini.race[target_company][good] = 1;
+				unit = new TTRPG_stats("astartes", target_company, good, "scout");
+				break;
+			*/
+			/* TODO - add "neophyte" in scr_marine_struct
+			case "Neophyte":
+				obj_ini.wep1[target_company][good] = "Bolt Pistol";
+				obj_ini.wep2[target_company][good] = "Combat Knife";
+				obj_ini.armour[target_company][good] = "Scout Armour";
+				obj_ini.race[target_company][good] = 1;
+				unit = new TTRPG_stats("astartes", target_company, good, "scout");
+				break;
+			*/
+			/* TODO - add "serf" and the related stuff
+			case "Serf":
+				obj_ini.wep1[target_company][good] = "Bolt Pistol";
+				obj_ini.wep2[target_company][good] = "Combat Knife";
+				obj_ini.armour[target_company][good] = "Scout Armour";
+				obj_ini.race[target_company][good] = 1;
+				unit = new TTRPG_stats("astartes", target_company, good, "scout");
+				break;
+			*/
+		// Bit funny that CSMs are not 1.5...
 		// Faction 2 - Homo Sapiens Imperialis
+			/* TODO - add regular human stats, in scr_marine_struct
+			case "Mercenary":
+				obj_ini.wep1[target_company][good] = "Hellgun"; TODO - consider weapon options
+				obj_ini.wep2[target_company][good] = "Combat Knife";
+				obj_ini.armour[target_company][good] = "Scout Armour"; TODO - add imperial armour
+				obj_ini.experience[target_company][good] = 10;
+				obj_ini.race[target_company][good] = 2;
+				unit = new TTRPG_stats("astartes", target_company, good, "scout"); TODO - add regular human stat lines
+				break;
+			*/
+			// Might want to add Kasrkin and Ogryn hireling types
+		// Faction 2.5 - Renegades
+			/*
+			case "Auxiliary Soldier":
+				obj_ini.wep1[target_company][good] = "Hellgun"; TODO - consider weapon options
+				obj_ini.wep2[target_company][good] = "Combat Knife";
+				obj_ini.armour[target_company][good] = "Scout Armour"; TODO - add imperial armour
+				obj_ini.experience[target_company][good] = 10;
+				obj_ini.race[target_company][good] = 2.5;
+				unit = new TTRPG_stats("renegades", target_company, good, "human"); TODO - add proper stat lines
+				break;
+			*/
 		// Faction 3 - Adeptus Mechanicus
 			switch (man_role) {
 			case "Skitarii":
 				obj_ini.wep1[target_company][good] = "Hellgun";
-				obj_ini.wep2[target_company][good] = ""; // Consider giving the poor fellow a "Combat Knife" or other melee weapon
+				obj_ini.wep2[target_company][good] = "Combat Knife"; // I should consider implementing a "Light Combat Knife", which is human-sized
```

Comment:
I don't think this is necessary, as all units get a default melee weapon if they have none (think of rifle stocks, fists, claws, etc).
Additionally, the combat knife is like a human sized sword, it's not meant for small units.
And plus, mechanicus units often don't have any melee weapons together with ranged.

Reply:
Ok, I'll remove the knife then.

Reply:
And, knife is removed.